installer/bundle/bundle_skel_Linux.sh

```diff
@@ -332,11 +333,11 @@ remove_and_install()
 {
     check_if_pkg_is_installed apache-cimprov
     if [ $? -eq 0 ]; then
-        pkg_rm apache-cimprov force
+        pkg_rm apache-cimprov
```

Comment:
Currently the code is written to be "generic" (i.e. "docker-cimprov, apache-cimprov, mysql-cimprov").

This list doesn't change much; you could hard-code it, but you could also do something more generic based on what is installed (i.e. *-cimprov), processing the list. I'll leave that up to you.

Also be aware the SCX installs an uninstall script that can be run by the shell bundle to remove software. The intent was that the uninstall script (`/opt/microsoft/scx/bin/uninstall`) was the "latest/greatest" way to install software, and doing removal in the shell bundle was mostly for legacy versions if that file didn't exist.

So if you change the way to remove software, you should modify the uninstall script as well.